tests/ui/LHNItemsPresence.tsx

```diff
@@ -172,7 +191,131 @@ describe('SidebarLinksData', () => {
+            // When the SidebarLinks are rendered.
+            LHNTestUtils.getDefaultRenderedSidebarLinks();
+            const archivedReport: Report = {
+                ...createReport(false),
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                private_isArchived: DateUtils.getDBTime(),
+            };
+            const reportNameValuePairs = {
+                type: 'chat',
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                private_isArchived: true,
+            };
+
+            await initializeState({
+                [`${ONYXKEYS.COLLECTION.REPORT}${archivedReport.reportID}`]: archivedReport,
+            });
+
+            await waitForBatchedUpdatesWithAct();
+
+            // And the user is in the default mode
+            await Onyx.merge(ONYXKEYS.NVP_PRIORITY_MODE, CONST.PRIORITY_MODE.DEFAULT);
+            await Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS}${archivedReport.reportID}`, reportNameValuePairs);
+
+            // The report should appear in the sidebar because it's archived
+            expect(getOptionRows()).toHaveLength(1);
+        });
+
+        it('should display the selfDM report by default', async () => {
+            // When the SidebarLinks are rendered.
+            LHNTestUtils.getDefaultRenderedSidebarLinks();
+            const report = createReport(true, undefined, undefined, undefined, CONST.REPORT.CHAT_TYPE.SELF_DM, undefined);
+
+            // And the selfDM is initialized in Onyx
+            await initializeState({
+                [`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`]: report,
+            });
+
+            // The selfDM report should appear in the sidebar by default
+            expect(getOptionRows()).toHaveLength(1);
+        });
+
+        it('should display the unread report in the focus mode with the bold text', async () => {
+            // When the SidebarLinks are rendered.
+            LHNTestUtils.getDefaultRenderedSidebarLinks();
+            const report: Report = {
+                ...createReport(undefined, undefined, undefined, undefined, undefined, true),
+                lastMessageText: 'fake last message',
+                lastActorAccountID: TEST_USER_ACCOUNT_ID,
+            };
+
+            await initializeState({
+                [`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`]: report,
+            });
+
+            await waitForBatchedUpdatesWithAct();
+
+            // And the user is in focus mode
+            await Onyx.merge(ONYXKEYS.NVP_PRIORITY_MODE, CONST.PRIORITY_MODE.GSD);
+
+            // The report should appear in the sidebar because it's unread
+            expect(getOptionRows()).toHaveLength(1);
+
+            // And the text is bold
+            const displayNameText = getDisplayNames()?.at(0);
+            expect(displayNameText).toHaveStyle({fontWeight: FontUtils.fontWeight.bold});
         });
```

Comment:
Could you add that once you mark it as read, the same report is no longer visible?